src/ripple/net/impl/HTTPStream.cpp

```diff
@@ -65,7 +65,7 @@ SSLStream::connect(
     }
     catch (std::exception const& e)
     {
-        return fail(std::string("exception: ") + e.what());
+        return fail("exception");
```

Comment:
If `ec` isn't set, we'd end up with an error message that says "exception".  I recommend keeping the exception message.

Reply:
Done